TIGLViewer/src/TIGLViewerSettings.h

```diff
@@ -20,32 +20,48 @@
 #define TIGLVIEWERSETTINGS_H_
 
 #include <QColor>
+#include <QtCore/QObject>
+#include <Graphic3d_NameOfMaterial.hxx>
 
-class TIGLViewerSettings
+#include "ITIGLViewerSettingsChangedListener.h"
+
+class TIGLViewerSettings : public QObject
 {
+    Q_OBJECT
+
 public:
+    ~TIGLViewerSettings() override = default;
```

Comment:
Good, but see comment later (i.e. change is unrelated to the feature)

Reply:
As you mentioned there are many unrelated changes. Do you suggest moving them out of this PR?

Reply:
No. You can leave them. Maybe next time, it would be better to separate this.